src/test/java/org/prebid/server/analytics/reporter/greenbids/GreenbidsAnalyticsReporterTest.java

```diff
@@ -212,16 +222,24 @@ public void shouldReceiveValidResponseOnAuctionContextForVideo() throws IOExcept
                 .isEqualTo(HttpHeaderValues.APPLICATION_JSON.toString());
         assertThat(headersCaptor.getValue().get(HttpUtil.CONTENT_TYPE_HEADER))
                 .isEqualTo(HttpHeaderValues.APPLICATION_JSON.toString());
+        assertThat(headersCaptor.getValue().get(HttpUtil.USER_AGENT_HEADER))
+                .isEqualTo(givenUserAgent());
```

Comment:
let's extract a separate test that covers URL, headers and timeout since they are the same and don't depends on the imp type

in that way we don't need to duplicate assertions in each test